src/config/chain_config.js

```diff
@@ -13,11 +13,11 @@ class ChainConfig {
     this.auxiliaryChainId = process.env.ORIGIN_CHAIN_ID || 200;
     this.simpleTokenAddress = process.env.SIMPLE_TOKEN_ADDRESS || '0xca954C91BE676cBC4D5Ab5F624b37402E5f0d957';
     this.originMasterKey = process.env.ORIGIN_MASTER_KEY || '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116';
-    this.auxiliaryMasterKey = process.env.AUXILIARY_MASTER_KEY || '0x490cc731b076e58509e1f7cee6c68b39cfe5ad62';
+    this.auxiliaryMasterKey = process.env.AUXILIARY_MASTER_KEY || '0x12b38720245e7bf06428a433a34b90d77e0198e1';
     this.originBurnerAddress = process.env.ORIGIN_BURNER_ADDRESS || '0x0000000000000000000000000000000000000000';
     this.auxiliaryBurnerAddress = process.env.AUXILIARY_BURNER_ADDRESS || '0x0000000000000000000000000000000000000000';
     this.originDeployerAddress = process.env.ORIGIN_DEPLOYER_ADDRESS || '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116';
-    this.auxiliaryDeployerAddress = process.env.AUXILIARY_DEPLOYER_ADDRESS || '0x490cc731b076e58509e1f7cee6c68b39cfe5ad62';
+    this.auxiliaryDeployerAddress = process.env.AUXILIARY_DEPLOYER_ADDRESS || '0x12b38720245e7bf06428a433a34b90d77e0198e1';
```

Comment:
Why are you changing the defaults? You should not need to.

Reply:
Agree!. It's  by mistake 😞. Will update.